commands/credentials.js

```diff
@@ -51,11 +52,36 @@ function * run (context, heroku) {
         return renderAttachment(attachment, app, isLast)
       })
 
-      return [cred].concat(attLines).join('\n')
+      let rotationLines = []
+      let credentialStore = credentials.filter(a => a.name === cred)[0]
+      if (credentialStore.state === 'rotating') {
+        let formatted = credentialStore.credentials.map(function (credential, idx) {
+          return {
+            'user': credential.user,
+            'state': credential.state,
+            'connections': credential.connections
+          }
+        })
+        let connectionInformationAvailable = formatted.some(function (c) { return c.connections })
```

Comment:
Not a big deal, but I find something like `formatted.some((c) => c.connections)` more readable...